### PR TITLE
[Merged by Bors] - Default (nonces, threads) = (288, CPU*3/4) for Mainnet

### PR DESCRIPTION
**Pull request by poszu**

### CODE REVIEW

**Comment by poszu** [comment]
Perhaps we should set threads to all cores on low end CPU with not many cores?

**Review by pigmej — APPROVED** [review]

**Comment by pigmej** [comment]
It will be hard to predict and still some spare for the system needed imho:)

**Review by countvonzero — APPROVED** [review]

**Comment by pigmej** [comment]
bors merge

**Comment by fasmat** [comment]
bors merge

**Comment by countvonzero** [comment]
bors merge

File: config/presets/fastnet.go [hunk 1]
```diff
@@ -61,6 +61,8 @@ func fastnet() config.Config {
 	conf.SMESHING.Opts.Throttle = true
 	// Override proof of work flags to use light mode (less memory intensive)
 	conf.SMESHING.ProvingOpts.Flags = postCfg.RecommendedPowFlags()
+	conf.SMESHING.ProvingOpts.Nonces = 16
+	conf.SMESHING.ProvingOpts.Threads = 1
```
Comment (fasmat) [inline config/presets/fastnet.go]: These 2 shouldn't need to be set. Fastnet uses `config.DefaultConfig` as base, not `config.MainnetConfig()`.

**Comment by fasmat** [comment]
bors merge

**Comment by fasmat** [comment]
bors merge

**Pull request closed without merging**